framework/PageForm/Inputs/PageFormAsyncMultiSelect.tsx

```diff
@@ -40,6 +40,7 @@ export type PageFormAsyncMultiSelectProps<
   | 'onBrowse'
   | 'queryLabel'
   | 'compareOptionValues'
+  | 'validate'
```

Comment:
PageFormAsyncMultiSelect has a `validate` prop defined on line 25. If we remove it from PageAsyncMultiSelect then we should remove it here as well.